app/src/main/java/se/rebeccazadig/bokholken/Adverts/Adverts.kt

```diff
@@ -1,11 +1,11 @@
 package se.rebeccazadig.bokholken.Adverts
 
 data class Adverts(
-    val bokTitel: String? = null,
-    val bokForfattare: String? = null,
+    val bookTitle: String? = null,
```

Comment:
1) just "title" and "author" instead of bookTitle and bookAuthor? 
2) Do you want to be able to create an advert without a title? if no - it should not be nullable. Same for other properties.